pkg/arguments/interactive.go

```diff
@@ -280,12 +281,31 @@ func PromptFilePath(fs *pflag.FlagSet, flagName string, required bool) error {
 			if err != nil {
 				return err
 			}
+			response, err = resolveRelativePath(response)
+			if err != nil {
+				return err
+			}
 			fs.Set(flagName, response)
 		}
 		return nil
 	})
 }
 
+// Golang does not support tilde file paths https://github.com/golang/go/issues/57569
+// However, we try to resolve this by manually so user can proceed further
+func resolveRelativePath(path string) (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+	dir := usr.HomeDir
+	absPath := dir
+	if strings.HasPrefix(path, "~/") && len(path) > 2 {
+		absPath = filepath.Join(dir, path[2:])
+	}
```

Comment:
how about a direct string replacement?

As in:

`strings.Replace(path, "~", usr.HomeDir, 1)`

Reply:
IMO, that would be an user input error and there's no much we could do anyway - unless I am missing something. 

Can you explain/provide a valid user case like that?

Reply:
Linux allows creating folders with a tilde sign in between. The user may have the service account file somewhere in the folder, and we should not restrict them from using that. With this change, we will be confusing the user with file not specified error.

Reply:
I see. Thanks for the example. 

I think we would be still covered with my suggested code, since only one replacement would be done:

```
	if strings.HasPrefix(path, "~/") {
		absPath = strings.Replace(path, "~", usr.HomeDir, 1)
	}
```

wdyt?

Reply:
Oh you mean keeping the if condition and then replacing! The UI showed me all the lines in the comment for some reason so I thought you suggested replacing all of it. 
But this change works. Updated the code.